ci/azure-build-release.yml

```diff
@@ -0,0 +1,93 @@
+steps:
+- checkout: self
+  submodules: true
+
+- template: azure-install-rust.yml
+
+- bash: echo "##vso[task.setvariable variable=RUSTC_VERSION;]`rustc --version`"
+  displayName: Set rustc version string for caching
+
+# - bash: |
+#     set -e
+#     curl -Lfo sccache.tar.gz https://github.com/mozilla/sccache/releases/download/0.2.9/sccache-0.2.9-x86_64-apple-darwin.tar.gz
+#     tar -xzf sccache.tar.gz
+#     cp sccache-*/sccache /usr/local/bin/
+#   displayName: Install sccache (OSX)
+#   condition: and(succeeded(), eq(variables['Agent.OS'], 'Darwin'))
+
+# - bash: |
+#     set -e
+#     curl -Lfo sccache.tar.gz https://github.com/mozilla/sccache/releases/download/0.2.9/sccache-0.2.9-x86_64-unknown-linux-musl.tar.gz
+#     tar -xzf sccache.tar.gz
+#     sudo cp sccache-*/sccache /usr/local/bin/
+#   displayName: Install sccache (Linux)
+#   condition: and(succeeded(), eq(variables['Agent.OS'], 'Linux'))
+
+# - script: |
```

Comment:
nit: commented block here, should we remove it?

Reply:
Removed.